src/utils/__tests__/changelog.test.ts

```diff
@@ -552,11 +552,13 @@ describe('generateChangesetFromGit', () => {
       [
         '### Better drivetrain',
         '',
-        'By: @alice (#123), @bob (#456)',
+        '- Upgraded the manifold (#123) by @alice in [#123](https://github.com/test-owner/test-repo/pull/123)',
```

Comment:
We should strip the PR no at the end of commit logs or PR titles when they appear like this: `(#<PR_NO>)`